### ISSUE TITLE
Can there be another way the hours and days change from phone?

### ISSUE THREAD

**Issue by Anthrwpos** [Helioviewer-Project/helioviewer.org#475]
1. When I try to change the time and day on the page from mobile it's very difficult and I have to press the arrows (up and down), it can't be done manually through the number box.
Thanks

**Comment by Anthrwpos** [comment]
Any answer about it?

**Comment by dgarciabriseno** [comment]
I'll see about having it use the native phone input controls.

**Comment by Anthrwpos** [comment]
> I'll see about having it use the native phone input controls.

Hi, is there a way when i select the hours or minutes from mobile to show the keyboard instead of doing it manually with the up and down arrows? 

Thanks

**Comment by dgarciabriseno** [comment]
@Anthrwpos can you go to https://beta.helioviewer.org on mobile and let me know if that works better?

**Comment by Anthrwpos** [comment]
> @Anthrwpos can you go to https://beta.helioviewer.org on mobile and let me know if that works better?

Hi. 

This is a much easier way to write the date at least in my case. Now when i select the date in the dark box, the mobile keyboard appears.  
The change will also take place on the main page of helioviewer?
Thank you.

**Comment by dgarciabriseno** [comment]
That's good. I think it works better in general, the arrows were really small to press too.
I wanted to use the native time picker that's built in to mobile devices, but browsers don't let you specify that you want a 24 hour clock and insist on showing AM/PM.
But I think this works okay. I'll merge it over to the main helioviewer.

**Comment by Anthrwpos** [comment]
> That's good. I think it works better in general, the arrows were really small to press too.
> I wanted to use the native time picker that's built in to mobile devices, but browsers don't let you specify that you want a 24 hour clock and insist on showing AM/PM.
> But I think this works okay. I'll merge it over to the main helioviewer.

Another thing i noticed when i select the hours in the time box the keyboard appears and i change the time but when i select the minutes after that (or when I pelress "the enter arrow➡️", the keyboard disappears and i have to select the minutes again to get the keyboard to appear again.
Thanks

**Comment by Anthrwpos** [comment]
I thought the arrows wouldn't be removed.  is it easy to have the time change option arrows too?

Thanks

**Comment by dgarciabriseno** [comment]
Sorry about that, I can add the arrows back. I removed them because I felt they would be in the way if the primary input was with the keyboard.

What kind of phone are you using? On mine, the keyboard stays open so I didn't have that problem. I'll have to be able to run into the same thing to be able to fix it.

**Comment by Anthrwpos** [comment]
> Sorry about that, I can add the arrows back. I removed them because I felt they would be in the way if the primary input was with the keyboard.
> 
> What kind of phone are you using? On mine, the keyboard stays open so I didn't have that problem. I'll have to be able to run into the same thing to be able to fix it.

You mean when you select for example the hours and change the numbers and then select the minutes, the keyboard doesn't disappear. I've tried it with Opera and Chrome.  I emphasize that the keyboard that appears is only with numbers.

If you are allowed, what mobile phone do you have?  I have a xiaomi.  I guess this issue has nothing to do with the Android versionm right?

Thanks

**Comment by dgarciabriseno** [comment]
Ah ok. I see it. This is what I get for only testing on firefox 🤦🏽‍♂️
You're right, android version doesn't matter. Browser & version is what matters.

**Comment by dgarciabriseno** [comment]
The keyboard should not close anymore.

**Comment by Anthrwpos** [comment]
> Ah ok. I see it. This is what I get for only testing on firefox 🤦🏽‍♂️
> You're right, android version doesn't matter. Browser & version is what matters.

Ok, thanks.

"And this is what I get for only testing on Firefox". You have added any image or is only text? Because as you write it I think there is also a picture I need to see, unless I'm wrong

**Comment by dgarciabriseno** [comment]
No, just text. It's an expression. I only tested on firefox so I didn't notice the issue of the keyboard disappearing on other browsers.
